ArmoniK.SDK.Client.Test/CMakeLists.txt

```diff
@@ -1,31 +1,63 @@
 cmake_minimum_required(VERSION 3.22)
 set(PROJECT_NAME ArmoniK.SDK.Client.Test)
 set(NAMESPACE ArmoniK::Sdk::Client::Test)
+set(version 0.1.0)
```

Comment:
I'd prefer it to be a variable to be set by the parent CMakeList (and give it as -DVERSION for example)